hw/dv/tools/xcelium/unr.cfg

```diff
@@ -0,0 +1,13 @@
+check_unr -setup 
```

Comment:
Please add the copyright banner to this file.

Reply:
yes fixed